### DISCUSSION TITLE
如何修改论文词条默认打开的PDF附件

### DISCUSSION THREAD

**Discussion by JSYUs** [zotero-chinese/wiki#167] [求助 / Q&A] [answered]
我有一个论文词条, 一开始该词条下只有一个PDF附件. 最近, 我更新了该论文的最新版本, 因此, 该词条出现了第二个PDF附件. 但是, 当我双击该论文词条的时候, 它总是默认打开old PDF附件. 因此, 如何修改论文词条默认打开的PDF附件, thanks.

**Comment by northword** [comment, marked as answer]
此功能尚未实现，关注：https://github.com/zotero/zotero/pull/3333

**Comment by jiaojiaodubai** [comment]
Zotero确实将较旧的附件做为默认打开的附件，你可以尝试移出所有附件，再依次重新添加使得你想打开的打一个附件是较早拖入的那个

**Comment by Maxpicca-Li** [comment]
我有一计：我发现 Zotero 默认打开文件是按照附件 token（姑且这么叫吧，就是附件所在的文件夹名词）的字符序最小的一个，所以你只需要把两个附件文件夹内的所有文件进行对调，然后重启 Zotero 即可。
